gst_plugins/python/savant_rs_video_decode_bin.py

```diff
@@ -390,6 +422,9 @@ def build_decoder(self, branch: BranchInfo):
         )
```

Comment:
As far as I understand, it can theoretically contain queues. [Docs ](https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/main/subprojects/gst-plugins-base/gst/playback/gstdecodebin2.c?ref_type=heads#L45-63)

You think we can safely ignore queue related `decodebin` properties such as [max-size-buffers](https://gstreamer.freedesktop.org/documentation/playback/decodebin.html?gi-language=c#decodebin:max-size-buffers)?

Reply:
I see, `decodebin` has `max-size-` properties for the queues. I'll make `savant_rs_decode_bin` to set these properties.